nixos/modules/system/boot/stage-1.nix

```diff
@@ -225,17 +207,7 @@ let
         # Make sure that the patchelf'ed binaries still work.
         echo "testing patched programs..."
         $out/bin/ash -c 'echo hello world' | grep "hello world"
-        ${
-          if zfsRequiresMountHelper then
-            ''
-              $out/bin/mount -V 1>&1 | grep -q "mount from util-linux"
-              $out/bin/mount.zfs -h 2>&1 | grep -q "Usage: mount.zfs"
-            ''
-          else
-            ''
-              $out/bin/mount --help 2>&1 | grep -q "BusyBox"
-            ''
-        }
+        $out/bin/mount -V 1>&1 | grep -q "mount from util-linux"
```

Comment:
That redirection looks wonky. Is that intended or should it have been `2>&1` instead, to redirect stderr to stdout?